.github/workflows/pr-test-e2e.yml

```diff
@@ -317,3 +337,24 @@ jobs:
           sleep 10
           curl http://localhost:8080/metrics
           kill %
+          
+      - name: Dump redacted Secret YAMLs
+        if: always()
+        run: |
+          SECRETS_LIST=$(kubectl --context kind-kubeflex get secrets -A -o jsonpath='{range .items[*]}{.metadata.namespace}{" "}{.metadata.name}{"\n"}{end}')
+          if [ -z "$SECRETS_LIST" ]; then
+            echo "No secrets found."
+            exit 0
+          fi
+          echo "$SECRETS_LIST" | while read -r ns name; do
+            echo "---"
+            SECRET_JSON=$(kubectl --context kind-kubeflex get secret "$name" -n "$ns" -o json)
+            # Print the Secret's metadata, deleting both data fields as suggested.
+            echo "$SECRET_JSON" | jq 'del(.data) | del(.stringData)' | yq -P
+            # Combine both .data and .stringData fields, then check if the result is a non-empty object.
+            COMBINED_DATA=$(echo "$SECRET_JSON" | jq '.data + .stringData | select(. != null and . != {})')
+            if [ -n "$COMBINED_DATA" ]; then
+              # Use the more succinct `map_values` function to redact all keys, then format as YAML.
+              echo "$COMBINED_DATA" | jq '{"data": map_values("<redacted>")}' | yq -P
+            fi
+          done
```

Comment:
Not loving the alert here.
<img width="197" height="57" alt="image" src="https://github.com/user-attachments/assets/abbcd934-9386-4159-9769-48fb47eae480" />